x-pack/elastic-agent/_meta/config/providers.yml.tmpl

```diff
@@ -0,0 +1,42 @@
+# Providers
+
+# Providers supply the key/values pairs that are used for variable substitution
+# and conditionals. Each provider's keys are automatically prefixed with the name
+# of the provider.
+
+#providers:
+
+# Agent provides information about the running agent.
+#  agent:
+#    enabled: true
+
+# Docker provides inventory information from Docker.
+#  docker:
+#    enabled: false
```

Comment:
Why `enabled: false`?

Reply:
When I've created it I though it was making more sense to set it to _false_, but after your comment and a weekend I think I should use true.